### PR TITLE
docs - add gpfdist support for writing gzip compressed files.

### PR DESCRIPTION
**Pull request by mkiyama**
Also added links to the gpfdist utility in other topics.

Will be backported to 5X_STABLE

### CODE REVIEW

**Comment by mkiyama** [comment]
@lij55 - please let me know if you are not the correct person to review.
thanks

File: gpdb-doc/dita/utility_guide/admin_utilities/gpfdist.xml [hunk 1]
```diff
@@ -43,10 +43,12 @@
           <codeph>INSERT</codeph> into the external table, and writes to an output file.</p>
       <p>For readable external tables, if load files are compressed using <codeph>gzip</codeph> or
           <codeph>bzip2</codeph> (have a <codeph>.gz</codeph> or <codeph>.bz2</codeph> file
-        extension), <codeph>gpfdist</codeph> uncompresses the files automatically before loading
-        provided that <codeph>gunzip</codeph> or <codeph>bunzip2</codeph> is in your path. </p>
-      <note type="note">Currently, readable external tables do not support compression on Windows
-        platforms, and writable external tables do not support compression on any platforms.</note>
+        extension), <codeph>gpfdist</codeph> uncompresses the files before loading. For writable
+        external tables, <codeph>gpfdist</codeph> compresses the data using <codeph>gzip</codeph> if
```
Comment (lisakowen) [inline gpdb-doc/dita/utility_guide/admin_utilities/gpfdist.xml]: write compression is supported only for .gz?  (same comment on the gpfdist protocol page.)

File: gpdb-doc/dita/utility_guide/admin_utilities/gpfdist.xml [hunk 2]
```diff
@@ -43,10 +43,12 @@
...
+        the target file has a <codeph>.gz</codeph> extension. To compress or uncompress data
+          <codeph>gunzip</codeph> or <codeph>bunzip2</codeph> must in your path.</p>
```
Comment (lisakowen) [inline gpdb-doc/dita/utility_guide/admin_utilities/gpfdist.xml]: must these be in the $PATH - i.e. does gpfdist call these programs directly?  or does there need to be a related library on the external system?

File: gpdb-doc/dita/utility_guide/admin_utilities/gpfdist.xml (hunk 1 above)
Comment (mkiyama) [reply gpdb-doc/dita/utility_guide/admin_utilities/gpfdist.xml]: yup. specifying a file with a bz2 extension returns a message saying bzip is not supported.

File: gpdb-doc/dita/utility_guide/admin_utilities/gpfdist.xml (hunk 2 above)
Comment (mkiyama) [reply gpdb-doc/dita/utility_guide/admin_utilities/gpfdist.xml]: I think it does call them directly.

@lij55 - Would you be the person to answer Lisa's question?

Thanks

**Comment by lij55** [comment]
Sorry for late reply.  From the code and my test result,
- gpfidst need to link libz to support gz format. It support both reading and writing for .gz format
- gpfdist only support to reading for bz2. libbz2 seems to be static linked.

Neither gzip or bzip2 command is needed.

File: gpdb-doc/dita/utility_guide/admin_utilities/gpfdist.xml (hunk 2 above)
Comment (lij55) [reply gpdb-doc/dita/utility_guide/admin_utilities/gpfdist.xml]: gunzip or bunzip2 is not needed.

**Comment by mkiyama** [comment]
@lij55 Thanks.

**Pull request merged**